sqlstore/src/main/resources/db/migration/V8123__SequencerPartitionContainer.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE SequencerPartitionContainer ADD COLUMN sequencingParametersId bigint,
+ADD CONSTRAINT `FK_Spc_Sp` FOREIGN KEY
+(sequencingParametersId) REFERENCES SequencingParameters
+(parametersId);
+ALTER TABLE SequencingParameters ADD COLUMN movieTime bigint;
```

Comment:
`int` or even `mediumint` should be more than sufficient. Could even be unsigned to double the range since it can't be negative